src/auth/src/token.rs

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::credentials::Result;
```

Comment:
This suggests that maybe `Result` should be at the `crate::` level.

Reply:
Good idea, let's start with this while I work on the other stuff. #457